tests/api/test_api.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+To test "import peekingduck" python API mode
+"""
+
+import re
+import subprocess
+from pathlib import Path
+import pytest
+
+
+@pytest.mark.api
+@pytest.mark.usefixtures("tmp_dir")
+class TestApi:
+    @staticmethod
+    def exec_code(code: str) -> str:
+        """Helper to execute given code string
+
+        Args:
+            code (str): python program to run
+
+        Returns:
+            str: output from running given code
+        """
+        temp_dir = Path.cwd()
+        with open("tmp.py", "w") as f:
+            f.writelines(code)
+        cmd = ["python", "tmp.py"]
+        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, cwd=temp_dir)
+        out, err = proc.communicate()
+        assert err is None  # no errors at all
+        out_str = out.decode("utf-8")
+        return out_str
+
+    def test_version(self):
+        """
+        Check we get a valid version number
+        """
+        code = """
+from peekingduck import __version__ as ver
+print(ver)
+        """
```

Comment:
Consider using [`textwrap.dedent()`](https://github.com/aimakerspace/PeekingDuck/blob/dev/peekingduck/pipeline/pipeline.py#L64) to maintain more natural looking indentation

Reply:
Beautified text with `textwrap.dedent()`.